portfolio/src/main/webapp/js/customscript.js

```diff
@@ -26,3 +26,27 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// adds Hello Natalie! to the page
```

Comment:
Some convention. Same for the comments below.

```suggestion
// Adds "Hello Natalie!" to the page.
```